src/api/routes/application.js

```diff
@@ -12,15 +11,14 @@ module.exports = (app) => {
     /**
      * Creates a new Company Application
      */
-    router.post("/", validators.create, async (req, res) => {
+    router.post("/", validators.create, ...validators.businessRules, async (req, res, next) => {
```

Comment:
You're checking application uniqueness twice, once in the create validator and another in the business rules. I would think it would be better if it was kept only in the create validator since that rule is going to be specific to this endpoint.